src/forms/subscriptions.ts

```diff
@@ -222,17 +223,17 @@ class FormFields extends BaseFormFields {
     }
 
     // getAnyConditions returns an array of Zendesk ANY trigger conditions
-    getAnyConditions(): any[] {
+    getAnyConditions(): ZDTriggerCondition[] {
         if (this.getSelectedSubTrigger() && this.getSelectedSubTrigger().conditions) {
             return this.getSelectedSubTrigger().conditions.any;
         }
         return [];
     }
 
     // getAnyConditions returns an array of Zendesk ALL trigger conditions
-    getAllConditions(): any[] {
+    getAllConditions(): ZDTriggerCondition[] {
         if (this.getSelectedSubTrigger() && this.getSelectedSubTrigger().conditions) {
-            return this.getSelectedSubTrigger().conditions.all;
+            return this.getSelectedSubTrigger().conditions.all as ZDTriggerCondition[];
```

Comment:
Why shouldn't the caller deal with the case that `conditions.all` is `undefined´?

Reply:
done.